src/labels/CustomLabels.labels

```diff
@@ -5495,6 +5495,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Error</shortDescription>
         <value>Error</value>
     </labels>
+    <labels>
+        <fullName>commonErrorAccess</fullName>
+        <categories>Common</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error message about permissions</shortDescription>
```

Comment:
```suggestion
        <shortDescription>Page-level error message about permissions to use a feature.</shortDescription>
```

Reply:
@psadie-sfdo added a little more information to the shortDescription to make it clear the error is talking about NPSP feature access. I'm having a hard time coming up with what else we'd be talking about for permissions, but seemed perhaps important to clarify. Feel free to reject my suggestion.